docroot/modules/custom/foia_file/foia_file.drush.inc

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @file
+ * FOIA Api module drush commands.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function foia_file_drush_command() {
+  $items = [];
+
+  $items['file-entity-updater'] = [
+    'description' => 'Updates file entities virus scan statuses.',
+    'core' => ['8+'],
+    'arguments' => [
+      'fully_qualified_fine_name' => 'A filename that includes the complete absolute path on the server.',
```

Comment:
Typo: should be `file_name` instead of `fine_name`

Reply:
Fine catch, 😄  I ended up pulling that out in a subsequent commit and just passing in the entire output of the scan results lines.